io.github.rinigus.OSMScoutServer.json

```diff
@@ -0,0 +1,255 @@
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://fallabs.com/kyotocabinet/pkg/kyotocabinet-1.2.77.tar.gz",
+                    "sha256": "56899329384cc6f0f1f8aa3f1b41001071ca99c1d79225086a7f3575c0209de6"
+                }
+            ]
+        },
+        {
+            "name": "microhttp",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ftp.gnu.org/gnu/libmicrohttpd/libmicrohttpd-0.9.59.tar.gz",
+                    "sha256": "9b9ccd7d0b11b0e179f1f58dc2caa3e0c62c8609e1e1dc7dcaadf941b67d923c"
+                }
+            ]
+        },
+        {
+            "name": "proj",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.osgeo.org/proj/proj-5.2.0.tar.gz",
+                    "sha256": "ef919499ffbc62a4aae2659a55e2b25ff09cccbbe230656ba71c6224056c7e60"
+                }
+            ]
+        },
+        {
+            "name": "mapnik",
+            "buildsystem": "simple",
+            "build-commands": [
+                "make reset",
+                "./configure BOOST_INCLUDES=/app/include BOOST_LIBS=/app/lib PROJ_INCLUDES=/app/include PROJ_LIBS=/app/lib INPUT_PLUGINS=\"sqlite,shape\" PREFIX=/app LINKING=shared OPTIMIZATION=2 CPP_TESTS=no CAIRO=no PLUGIN_LINKING=static MEMORY_MAPPED_FILE=no DEMO=no MAPNIK_INDEX=no MAPNIK_RENDER=no",
+                "make install",
+                "cp -r deps/mapbox/variant/include/mapbox /app/include"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/mapnik/mapnik/releases/download/v3.0.21/mapnik-v3.0.21.tar.bz2",
+                    "sha256": "6db7918e8fd24346dfc81745e455c383a718335ffa64015cf625fed5ed92b524"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/mapnik/mapnik.twkb.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/mapnik/mapnik.issue3384.patch"
+                }
+            ]
+        },
+        {
+            "name": "osmscout-server-fonts",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir /app/share/osmscout-server-fonts",
+                "cp -r fonts /app/share/osmscout-server-fonts"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/rinigus/osmscout-server-fonts.git",
+                    "commit": "9d550dc777695c4f731ac854aaec91bee79efde0"
+                }
+            ]
+        },
+        {
+            "name": "osmscout-server",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ln -s pro/osmscout-server_qtcontrols.pro . || true",
```

Comment:
There is `qmake` build system since a while. IIRC `shell` type runs right after source extraction.

Reply:
sure. But then 
* I will save 2 lines (qmake and make) 
* will have to add few lines for CONFIG
* add `ln` and mkdir & install lines somewhere else

That way its at least not spread all over